libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -1315,7 +1315,8 @@ void NavEKF2::setTouchdownExpected(bool val)
 
 // Set to true if the terrain underneath is stable enough to be used as a height reference
 // in combination with a range finder. Set to false if the terrain underneath the vehicle
-// cannot be used as a height reference
+// cannot be used as a height reference. Use to prevent range finder operation otherwise
+// enabled by the combination of EK2_RNG_AID_HGT and EK2_RNG_USE_SPD parameters.
```

Comment:
I don't actually see these EKx_RNG_AID_HGT and EKx_RNG_USE_SPD parameters anywhere..

Reply:
I've put a couple of commits at the end of my version of this PR here: https://github.com/rmackay9/rmackay9-ardupilot/commits/prise-ekf23-fix-hgt-sensor-select